rcl/test/rcl/test_graph.cpp

```diff
@@ -1295,6 +1295,32 @@ TEST_F(CLASSNAME(TestGraphFixture, RMW_IMPLEMENTATION), test_graph_guard_conditi
   ASSERT_GE(graph_changes_count, 4ul);
 }
 
+/* Test the graph when guard condition is triggered.
+ */
+TEST_F(CLASSNAME(TestGraphFixture, RMW_IMPLEMENTATION), test_graph_guard_condition_triggered) {
+  rcl_ret_t ret;
+  int64_t wait_time_out = -1;
```

Comment:
```suggestion
  const int64_t wait_time_out = -1;
```

Or consider getting rid of it completely, since it never changes.